spec/lib/rex/post/mssql/ui/console/command_dispatcher/core_spec.rb

```diff
@@ -18,6 +21,7 @@
 
   before(:each) do
     allow(client).to receive(:sock).and_return(rstream)
+    allow(client).to receive(:mssql_query).and_return(query_result)
```

Comment:
I think it'd be good to lock this down to the specific query that we're wanting to mock 👀 

```
    allow(client).to receive(:mssql_query).with(...).and_return(query_result)
```

Just so we don't sneak in accidental queries here in the future without realizing